styles/myStory.css

```diff
@@ -0,0 +1,65 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  background-image: url("/images/image8.jpg");
+  background-size: cover;
+}
+
+.overlay-text {
+  position: absolute;
+  color: white;
+  font-family: var(--paragraph-sans);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.overlay-text p {
+  margin-bottom: 50px;
+  margin: 0px;
+  text-transform: uppercase;
+  text-align: center;
+}
+.text-header {
+  font-size: 68px;
+}
+.text-description {
+  padding-top: 20px;
+  font-size: 16px;
+}
+.color-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+
+.myStory-content-wrapper {
+  background-color: white;
+}
+.myStory-inner-content {
+  padding: 40px 20vw;
```

Comment:
This doesn't guarantee max-width to 1500.

Reply:
I added max-width to the wrapper div of the inner-content

Reply:
and removed margin: 0 auto for the inner-content as it has padding with % values: the20vw (viewport width) padding on the left and right so it already centers the layout without using margin: 0 auto.